Wireframe/css/style.css

```diff
@@ -0,0 +1,128 @@
+:root{
+    --background-color: rgb(43, 42, 42);
+    --text-color: white;
+    --link-color: rgb(64, 126, 226);
+    --article_background:rgb(87, 85, 85);
+
+    
+
+}
+html{
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+body{
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 5px 0 5px 5px;
+
+    background-color: var(--background-color);
```

Comment:
When you ran Lighthouse, what did it tell you about the text contrast?

Reply:
Thank you, Sally, for your feedback and guidance. I acknowledge that the work I submitted was not satisfying to myself either. I will be working on making the requested changes. Additionally, I am aware that my code is not clean and tidy, and I am still struggling to fix this issue with my code.

Reply:
It's such a great idea to refactor the code in a proper and professional way. It could be helpful for myself first and then for the rest of the class, as we would become aware of common mistakes that amateurs make. I'd be grateful if you could do it.